config/locales/en.yml

```diff
@@ -1,5 +1,31 @@
-# Sample localization file for English. Add more files in this directory for other locales.
-# See https://github.com/svenfuchs/rails-i18n/tree/master/rails%2Flocale for starting points.
-
 en:
-  hello: "Hello world"
+  step_i_of_n: "Step %{i} of %{n}"
+  previous: Previous
+  next: Next
+  abort: Save for later
+  previous_questionnaire: Previous questionnaire
+  done: Done
+  print_answers: Print answers
+  questionnaire_not_completed:
+    heading: "This questionnaire hasn't been filled out completely!"
```

Comment:
Waar baseer je het op dat contracties minder net zijn dan volledig uitgeschreven zinnen? Ik ben het oneens met dat het de leesbaarheid/"netheid" verhoogd, en [https://www.google.nl/webhp?sourceid=chrome-instant&ion=1&espv=2&ie=UTF-8#q=contractions%20formal%20writing](diverse bronnen) geven ook aan dat contracties prima kunnen in formele tekst.